.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,107 @@
+name: "Bug Report"
+description: Use this template to report a bug
+labels: ["bug"]
+body:
+  - type: markdown
+    id: example
+    attributes:
+      value: >
+        Thanks for reporting.
+        
+        - Make sure you are able to reproduce this issue on the latest released version of [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+        
+        - Please search the existing issues to see if there has been a similar issue filed.
+        
+        - If you want to discover useful tips on how to avoid common pitfalls and use the Azure CLI successfully, please refer to [Tips for using the Azure CLI successfully](https://learn.microsoft.com/en-us/cli/azure/use-cli-effectively).
+  - type: textarea
+    id: desc
+    attributes:
+      label: Describe the bug
+      description: >
+        Please describe the bug.
+      placeholder: >
+        A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: cmd
+    attributes:
+      label: Related command
+      description: Please provide the related commands and parameters with az {command} if you can, so that we can quickly route to the related team for help.
+      placeholder: e.g., `az vm create --image`
+    validations:
+      required: true
+  - type: textarea
+    id: error
+    attributes:
+      label: Errors
+      description: >
+        Please provide a clear error message here.
+      placeholder: >
+        A clear error message of the bug.
+    validations:
+      required: true
+  - type: textarea
+    id: output
+    attributes:
+      label: Issue script & Debug output
+      description: >
+        Append `--debug` to each command in the script and rerun the script and paste the debug log here
+        
+        **⚠ ATTENTION:** Be sure to remove any sensitive information that may be in the logs
+      placeholder: >
+        Please provide the script and debug output here.
+    validations:
+      required: true
+  - type: textarea
+    id: expect
+    attributes:
+      label: Expected behavior
+      description: >
+        Please describe the expected behavior here.
+      placeholder: >
+        A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: azure cli version
+      description: Please provide the first line of `az --version`.
+      placeholder: azure-cli                         2.48.1
```

Comment:
How about making `Environment Summary` required and omit this field.

Reply:
@jsntcy,
What do you think?
At the meeting we decided to only make the CLI version mandatory, and the Environment Summary is just an optional one.

Reply:
Is it easier for bot if we have a separate filed for the version of azure cli?

Reply:
In fact, it is almost the same, I still search for the `azure cli version` on the entire PR body.